include/glow/IR/IRBuilder.h

```diff
@@ -58,6 +58,11 @@ class IRBuilder {
                                    llvm::ArrayRef<size_t> dims, Value *src,
                                    llvm::StringRef name);
 
+  TensorViewInst *createTensorView(ElemKind elemKind,
```

Comment:
Do we need both forms of `createTensorView`? May be the new form with offsets can take offset as an optional parameter with default value (e.g. an empty set of offsets)?